ec-europa/eubfr-data-lake
# feat(meta-index): replace dynamodb with elasticsearch - EUBFR-104

To test it:
- deploy the resources (it creates a new ES index in dev-meta), the services and the demos
- open 1 of the demos
- check the file list: now it requests data from ES instead of the demo server (which fetched data from dynamo)
- upload a file
- check if the file appears in the file list
- open the file
- check if all the meta are correct
